bob-common/src/configs/node.rs

```diff
@@ -543,6 +543,8 @@ pub struct Node {
     bind_to_ip_address: Option<SocketAddr>,
     #[serde(default = "NodeConfig::default_holder_group_size")]
     holder_group_size: usize,
+    #[serde(default = "NodeConfig::default_max_sequential_errors")]
+    max_sequential_errors: usize,
```

Comment:
Please, place this parameter closer to other network parameters: after `check_interval`

Reply:
Also, add this parameter to `node.yaml` in config-examples